charts/apiclarity/templates/traffic_sources/tap/daemonset.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.trafficSource.tap.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  annotations:
+    deprecated.daemonset.template.generation: "1"
+  name: {{ include "apiclarity-taper.name" . }}
+  namespace: '{{ .Release.Namespace }}'
+  labels:
+    {{ include "apiclarity-taper.labels" . }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ include "apiclarity-taper.name" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "apiclarity-taper.name" . }}
+    spec:
+      serviceAccountName: {{ include "apiclarity-taper.name" . }}
+      containers:
+        - command:
+            - ./agent
+            - -i
+            - any
+            - --log-level
+            - {{ .Values.trafficSource.tap.logLevel }}
+          env:
+            - name: GOGC
+              value: "12800"
+            - name: UPSTREAM_TELEMETRY_HOST_NAME
+              value: '{{ include "apiclarity.name" . }}.{{ .Release.Namespace }}:9000'
```

Comment:
we dont want to define a service name instead of {{ include "apiclarity.name" . }}.{{ .Release.Namespace }}
meaning define "apiclarity-taper.upstream = {{ include "apiclarity.name" . }}.{{ .Release.Namespace }}